staging/regions-v3.json

```diff
@@ -24,6 +24,7 @@
         "paymentiOSAppStoreIdentifier": "1140553099", 
         "paymentiOSAppUrlScheme": "fb313213768708402HART", 
         "active": true, 
+        "travelBehaviorDataCollectionEnabled": true, 
```

Comment:
Let's add one more variable for enrolling subjects into study - at some point we may want data collection to continue for existing subjects but stop adding more subjects (e.g., if we run out of incentives).  Maybe `enrollParticipantsInStudy`?